src/components/common/Actions.vue

```diff
@@ -130,6 +130,11 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
     return (process.env.JEST_WORKER_ID !== undefined)
   }
 
+  /** Is True if the Back button should be displayed. */
+  get isShowBackBtn (): boolean {
+    return (this.getCurrentStep > 1)
+  }
```

Comment:
This was previously a store getter. I moved it here because it's actually specific to this component only.